strapi/strapi
# fixed issue 21342: HP logos are misaligned

### What does it do?

Fixed HP logos are misaligned

### Why is it needed?

To Fixed #21342 : The icons are allinged center, causing the misaligned issue .By aligned it to left its solved.

### How to test it?

- Login
- Go to Homepage
- Check Join the community section on the right of the page

### Related issue(s)/PR(s)

#21342
